include/dynd/kernels/apply.hpp

```diff
@@ -162,8 +152,7 @@ namespace nd {
       {
         if (val.get_type().get_type_id() == pointer_type_id) {
           m_val = val.f("dereference").as<T>();
-        }
-        else {
+        } else {
```

Comment:
This is a bit annoying - the clang-format embedded in MSVC is behaving differently than other clang-format embeddings for this case.